web/src/Overview.jsx

```diff
@@ -100,11 +100,28 @@ function Overview() {
     ));
   };
 
+  const RightActions = () => {
+    if (products.length === 1) {
+      return "";
+    }
+
+    return (
+      <Button
+        isSmall
+        variant="plain"
+        icon={<ModeEditIcon />}
+        aria-label="Change selected product"
+        onClick={() => navigate("products")}
+      />
+    );
+  };
+
```

Comment:
NP: Now that we have _useSoftware_, we could extract this out of the Overview component. I mean, similar to what was done with InstallButton.